### ISSUE TITLE
yet another initiative: haskell-tools

### ISSUE THREAD

**Issue by rvion** [haskell/haskell-ide-engine#216]
https://github.com/haskell-tools/haskell-tools

on hackage:

http://hackage.haskell.org/package/haskell-tools-demo-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-refactor-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-cli-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-ast-trf-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-prettyprint-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-ast-fromghc-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-ast-gen-0.2.0.0
http://hackage.haskell.org/package/haskell-tools-ast-0.2.0.0

**Comment by alanz** [comment]
Yes, I know about it. Can you add it to the docs, for things to integrate at some point?
